config/health_care_application.yml

```diff
@@ -0,0 +1,30 @@
+endpoints:
```

Comment:
I think we need to clean this up a bit. I.e. rather than carrying around all endpoint references here, refer to a single env variable set in application.yml, and then devops will set the env appropriately based on where we are deploying, developers can set it locally as needed. For better or worse, we set env=production for all of dev/staging/prod. 

Similary I think for consistency we'll probably want to collapse the trust_chains into single concatenated files and store them in devops.